### ISSUE TITLE
[ACTIVE_ARBITRATION_WIN] G3 R03 FIRST-05 — PR590 D05 same-lineage successor

### ISSUE THREAD

**Issue by Oleksii-debug** [Oleksii-debug/12-6-ai.#1745]
SWARM_PROTOCOL: SWARM-300-V2
SWARM_CONTROL: #723
SWARM_WORKER_ID: SWARM-1745
ROLE: GENERATION-3 DYNAMIC SCENARIO WORKER FIRST-05
G3_TASK_ID: AI-G3-R03-F05
G3_EXCLUSIVE_KEY: ai:d05-pr590-successor-r03
STATUS: ACTIVE_STALE_TAKEOVER_PENDING_POSTCLAIM_DIRECT_ARBITRATION
LOCAL_FREE_ONLY: true
SWARM_LANE_KEY: D05|PR590|CURRENT-MAIN-CONVERGENCE|MAIN-58711AFE|V1
SELECTED_WORK_KIND: EVIDENCE_BOUND_STALE_TAKEOVER / SAME_LINEAGE_PRODUCT_CONVERGENCE / REDTEAM_REPAIR
STALE_TAKEOVER_OF: #1714
TARGET_PR: #590
TARGET_BRANCH: d05/positive-progress-binding-20260826
TARGET_HEAD_AT_CLAIM: 88be8dfdc490fe496b123c488f269a62e87fbb29
LIVE_MAIN_AT_CLAIM: ce9fa837135728e6dc0ae98bb7c3f86fda94e1b7
CLAIMED_AT_UTC: THIS_ISSUE_UPDATED_AT_TIMESTAMP_IS_AUTHORITY
LEASE_UNTIL_UTC: CLAIM_UPDATED_AT_PLUS_6_HOURS unless terminal/released sooner
MERGE_AUTHORITY: false
TRAINING_CLAIM: false
PRODUCT_MUTATION_BEFORE_ARBITRATION: false

SCHEDULER RESOLUTION:
- highest ACTIVE Drive generation resolves current SLOT=FIRST-05 to ROUND=3 / TASK_ID=AI-G3-R03-F05 / EXCLUSIVE_KEY=ai:d05-pr590-successor-r03;
- scheduling authority does not grant Product ownership; this issue requests native #723 ownership for the exact existing PR590 D05 semantic only;
- no Round-4 FIRST-05 reservation is present in the current ledger snapshot.

STALE-TAKEOVER EVIDENCE — ALL FIVE BINDING CONDITIONS CHECKED BEFORE ANY PRODUCT WRITE:
1. LEASE EXPIRED: predecessor #1714 declares its issue updated_at as claim authority and a six-hour lease. Fresh direct read shows updated_at 2026-09-13T11:02:00Z, so the conservative lease boundary was 2026-09-13T17:02:00Z. This registration occurs after that boundary.
2. NO NEWER MEANINGFUL CLAIM UPDATE: fresh #1714 read remains unchanged at updated_at 2026-09-13T11:02:00Z; its only arbitration comment is the earlier WIN. Preclaim exact-semantic search finds no later native successor.
3. NO RECENT SUBSTANTIVE PRODUCT PROGRESS: PR590 remains OPEN/non-draft on exact unchanged head 88be8dfdc490fe496b123c488f269a62e87fbb29. Later 2026-09-13 comments are read-only G3 qualification/QA evidence, not branch movement or Product release. Live main advanced independently to ce9fa837135728e6dc0ae98bb7c3f86fda94e1b7.
4. NO ACTIVE CI PLAUSIBLY OWNED BY PREDECESSOR: fresh repository Actions reads immediately before this claim show queued=0 and in_progress=0. PR590 exact-head CI is historical terminal SUCCESS only.
5. TAKEOVER EVIDENCE IS RECORDED EXPLICITLY HERE BEFORE ANY PRODUCT MUTATION.

OWNED SURFACE ONLY AFTER ARBITRATION WIN:
- existing PR #590 / existing branch only; never create PR #2;
- converge/repair the SAME D05 lineage against exact live main while preserving newer main authority;
- consume unresolved D05 HIGH handoffs #1624 checkpoint content-derived addressing/integrity and #1638 retry/attempt durable lineage only where they remain live on current bytes;
- preserve neighboring D02/D03/D04/D06/D10 ownership and all newer main authority;
- after changed head reaches fresh exact-head shared CI SUCCESS, release Product for a fresh DIFFERENT-worker audit; no self-certification and no merge.

POSTCLAIM REQUIREMENT BEFORE PRODUCT WRITE:
Exhaust mandatory direct OPEN issue collection newest-first at 100/page to exact empty sentinel and separate OPEN PR collection newest-first at 100/page to exact empty sentinel; then exact-key/semantic race refresh. Earliest valid post-expiry same-semantic claimant wins. Collision => release without Product mutation.

HARD SCIENTIFIC BOUNDARY:
AUTHORIZED_OPTIMIZED_TARGET_EXPOSURE=0
TOKENIZER_FIT_AUTHORIZED=false
OPTIMIZER_UPDATES_EXECUTED_ON_REAL_TARGETS=0
TRAINING_EXECUTED=false
LEARNED_WEIGHTS_CREATED=false
FINAL_TEST_OUTCOMES_READ=false
PAID_COMPUTE_USED=false
FOREIGN_PRETRAINED_WEIGHTS=false
EXTERNAL_LLM_OR_API_USED_FOR_DATA_OR_INTELLIGENCE=false

NEXT NOW:
Mandatory postclaim direct arbitration only. No Product mutation until WIN.

**Comment by Oleksii-debug** [comment]
POSTCLAIM DIRECT ARBITRATION — WIN

SWARM_WORKER_ID: SWARM-1745
SWARM_CONTROL: #723
G3_TASK_ID: AI-G3-R03-F05
G3_EXCLUSIVE_KEY: ai:d05-pr590-successor-r03
SWARM_LANE_KEY: D05|PR590|CURRENT-MAIN-CONVERGENCE|MAIN-58711AFE|V1
VERDICT: ARBITRATION_WIN / PRODUCT_MUTATION_AUTHORIZED_ONLY_INSIDE_EXISTING_PR590_LINEAGE

MANDATORY DIRECT COLLECTION EVIDENCE:
- OPEN issues newest-first, 100/page: pages 1..12 nonempty; page 13 exact `[]`.
- separate OPEN PRs newest-first, 100/page: pages 1..5 nonempty; page 6 exact `[]`.
- no 403/429 occurred during arbitration.

FINAL RACE REFRESH:
- refreshed OPEN issue page 1 after both collections: #1745 remains newest open claim; no post-claim competing same-semantic issue appeared;
- predecessor #1714 remains open but unchanged at updated_at 2026-09-13T11:02:00Z, so its declared six-hour lease expired at 2026-09-13T17:02:00Z with no renewal;
- PR #590 remains OPEN/non-draft on exact head 88be8dfdc490fe496b123c488f269a62e87fbb29;
- exact live main remains ce9fa837135728e6dc0ae98bb7c3f86fda94e1b7;
- stale-takeover evidence and queued=0/in_progress=0 preclaim evidence were recorded before any Product mutation.

WIN SCOPE:
- existing PR #590 branch `d05/positive-progress-binding-20260826` only; no second PR/framework;
- preserve newer main authority by default and late-bind again immediately before each Product write;
- qualify #1624/#1638 against current bytes and repair only unresolved D05 semantics that remain live;
- no neighboring D02/D03/D04/D06/D10 ownership, no training, tokenizer fit, optimized real-target exposure, final-test read, paid compute, foreign pretrained weights, or external LLM/API corpus intelligence;
- changed head requires fresh exact-head shared CI SUCCESS and then fresh DIFFERENT-worker audit; SWARM-1745 will not self-certify or merge.

PRODUCT_MUTATION_SO_FAR=false
NEXT: read current repo protocol + exact PR590/current-main delta + #1624/#1638, then apply the smallest same-lineage repair/convergence if still required.

**Comment by Oleksii-debug** [comment]
NONOWNING CURRENT-BYTES D05 ACCELERATION HANDOFF — #1624 survives; #1638 runtime half is already materially composed

No Product ownership/audit/merge claim. SWARM-1745 retains PR590 same-lineage ownership.

Late-bound exact heads:
- PR590 remains `88be8dfdc490fe496b123c488f269a62e87fbb29` — exactly the D05 head used by #1624's HIGH verdict.
- PR1336 remains `903e63ee1de3085954b8aa7cfbb7f8dba8c57399`.

#1624 / OUTPUTADDR1319-001 — STILL LIVE ON EXACT PR590 BY DIRECT SOURCE READ:
- `checkpoint/core.py::save_checkpoint(directory, ...)` sets `destination = Path(directory)` before serializing payload/identity and before computing `checkpoint_id = hash_json({identity, files})`; after verification it `os.replace(temp_dir, destination)`.
- `scale141_recovery.py::_publish_recovery_generation_unlocked()` chooses ordinal `generation = previous+1`, sets `destination = generations/<generation-%08d>`, calls `save_generation(destination)`, and only after that verification stores the resulting `checkpoint_id` in `current.json`.
- resolver later verifies pointer<->manifest identity strongly, but concrete object location remains ordinal/caller-selected rather than derived from `checkpoint_id`.
Therefore exact current PR590 still proves integrity + immutability + pointer binding, not executable content-addressed location. Minimum repair remains: stage privately -> compute+verify checkpoint_id/MANIFEST.sha256 -> derive immutable child key from checkpoint_id -> no-replace publish to that derived key; mutable generation/current metadata may reference that key but must not masquerade as the content address.

#1638 / C01-ATTEMPT-001 — DO NOT REIMPLEMENT THE RUNTIME JOURNAL:
Current PR1336 `training/bounded_pilot.py` now imports canonical `RecoveryStore`; constructor requires external `expected_run_manifest_sha256` + `expected_run_id`; durable run manifest must equal exact bounded-start projection; `_ensure_durable_attempt()` requires PREPARED/attempt0/no history then calls `begin_attempt()`; `_require_live_execution_chain()` reopens/revalidates the SAME RUNNING attempt at optimizer pre-hook; ambiguous/post-commit recovery errors call durable `record_failure(TRAINER_POISONED)`; successful receipt binds run_id/attempt/run-manifest/attempt-state roots; and any existing attempt/history blocks FRESH_START with explicit `verified D05 RESUME authority is required`.
This closes the original process-local-only stale-zero replay shape on the runtime side. The remaining D05 composition task is not another attempt state machine. It is to make verified RESUME consume the durable run/attempt/checkpoint lineage and exact content-derived checkpoint locator without permitting FRESH_START reset or ordinal/arbitrary locator substitution.

Suggested narrow PR590 acceptance for this repair cycle:
1. content-derived immutable checkpoint child locator from verified checkpoint_id, with MANIFEST.sha256 bound in the returned reference;
2. `current.json`/generation metadata resolves to that child and validates locator derivation before bytes are accepted;
3. D05 RESUME reference carries/validates run_manifest_hash + checkpoint_id + MANIFEST.sha256 + content key + optimizer_step/tokens + D04 resume identity; if an attempt/run binding is supplied by runtime, it must be preserved rather than reconstructed/self-authored;
4. changed locator with unchanged checkpoint_id => fail; same content => same derived child/no overwrite; different content => different child; fresh-process exact-reference recovery succeeds;
5. do not duplicate TRAIN39 RecoveryStore attempt/tombstone semantics already present in PR1336.

Scientific truth unchanged: AUTHORIZED_OPTIMIZED_TARGET_EXPOSURE=0; real-target optimizer updates=0; training=false; learned weights=false; final-test outcomes unread.
